### DISCUSSION TITLE
边类型相关操作

### DISCUSSION THREAD

**Discussion by ShuSongXiao** [TuGraph-family/tugraph-db#763] [Q&A]
边类型创建好之后，设置好了起点类型和终点类型，如果发现起点类型和终点类型设置错误，能否删除已建好的设置关系，重新设置新的关系？

**Comment by ljcui** [comment]
参考 https://github.com/TuGraph-family/tugraph-db/blob/master/docs/zh-CN/source/development_guide.md，删除边类型然后重建。
